src/scripts/discord.js

```diff
@@ -25,22 +25,22 @@ const observer = (event, arg) => {
       rpc.setActivity({
         ...idleStatus,
         ...{
-          details: `Listening to ${mediaInfoModule.mediaInfo.title}`,
+          details: `${mediaInfoModule.mediaInfo.title}`,
           state: mediaInfoModule.mediaInfo.artist
             ? mediaInfoModule.mediaInfo.artist
             : "unknown artist(s)",
           startTimestamp: parseInt(now),
           endTimestamp: parseInt(remaining),
           largeImageKey: mediaInfoModule.mediaInfo.image,
           largeImageText: (mediaInfoModule.mediaInfo.album) ? mediaInfoModule.mediaInfo.album : `${idleStatus.largeImageText}`,
-          buttons: [{ label: "Play on Tidal", url: mediaInfoModule.mediaInfo.url }],
+          buttons: [{ label: "Listen Along", url: mediaInfoModule.mediaInfo.url }],
         },
       });
     } else {
       rpc.setActivity({
         ...idleStatus,
         ...{
-          details: `Watching ${mediaInfoModule.mediaInfo.title}`,
```

Comment:
I like this change (even though I didn't like the removal of "listening to").

Reply:
I generally think the differences between "listening to" and "watching" should be kept as both are completely different and if people view the status they would exactly know what you are doing compared to it just always showing the video title or song title

Reply:
Let's go with @Mar0xy 's suggestion here. I don't utilize the "watching" feature at all and am thus probably biased against it.